diff --git "a/C:\\Users\\endymion\\AppData\\Local\\Temp\\regen140-XWGHyx\\header-base.js" "b/C:\\Users\\endymion\\AppData\\Local\\Temp\\regen140-XWGHyx\\header-build.js"
index 49150f9..27c97b1 100644
--- a/better-xcloud.user.js
+++ b/better-xcloud.user.js
@@ -1,7 +1,7 @@
 // ==UserScript==
 // @name         Better xCloud
 // @namespace    https://github.com/redphx
-// @version      6.7.12-perf10
+// @version      1.4.0
 // @description  Improve Xbox Cloud Gaming (xCloud) experience
 // @author       redphx
 // @license      MIT
@@ -10,6 +10,6 @@
 // @exclude      https://www.xbox.com/*/xbox-game-pass/play-day-one
 // @run-at       document-start
 // @grant        none
-// @updateURL    https://raw.githubusercontent.com/redphx/better-xcloud/typescript/dist/better-xcloud.meta.js
-// @downloadURL  https://github.com/redphx/better-xcloud/releases/latest/download/better-xcloud.user.js
+// @updateURL    https://github.com/Endymi0n74/better-xcloud-perf/releases/latest/download/better-xcloud.meta.js
+// @downloadURL  https://github.com/Endymi0n74/better-xcloud-perf/releases/latest/download/better-xcloud.user.js
 // ==/UserScript==
